pallets/multisig/src/lib.rs

```diff
@@ -97,10 +97,14 @@ use polymesh_common_utilities::{
     transaction_payment::{CddAndFeeDetails, ChargeTxFee},
     Context,
 };
-use polymesh_primitives::{AuthorizationData, AuthorizationError, IdentityId, Signatory};
+use polymesh_primitives::{
+    AuthorizationData, AuthorizationError, IdentityId, PalletPermissions, Permissions, Signatory,
+};
 use sp_runtime::traits::{Dispatchable, Hash};
-use sp_std::{convert::TryFrom, prelude::*};
+use sp_std::{convert::TryFrom, iter, prelude::*};
+
 type Identity<T> = identity::Module<T>;
+type CallPermissions<T> = pallet_permissions::Module<T>;
 
```

Comment:
when a multisig dispatches a call, the call name, and module name in  `pallets/permissions` should be updated.